### PR TITLE
BAU: Add PR template

### PR DESCRIPTION
**Pull request by johnoneill46**
Add a pull request template to make pull requests more descriptive and easier to understand for reviewers.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_
+
+
+### Change description
+_A brief description of the pull request_ 
+
+- [ ] Unit tests and other appropriate tests added or updated
```
Comment (gio-karunakaran) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: is this applicable for this repo?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] README and other documentation has been updated / added (if needed)
+- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")
+
+
+### How to test
```
Comment (gio-karunakaran) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: applicable or can we remove?

**Review by gio-karunakaran — CHANGES_REQUESTED** [review]
some comments

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (johnoneill46) [reply .github/PULL_REQUEST_TEMPLATE.md]: Agree - changes made

**Pull request closed without merging**